api_backend.yaml

```diff
@@ -152,18 +152,21 @@ paths:
         "200":
           description: Found.
           schema:
-            $ref: "#/definitions/CreatedMessageWithContent"
+            $ref: "#/definitions/CreatedMessageWithContentAndAttachments"
           examples:
             application/json: |
-              content: {
-                markdown: "hey hey !! <a style=\"color: red\" href=\"http://example.com\"> some content here ..... this is a link with a style applied, some other content</a>",
-                subject: "my subject ............"
+              message: {
```

Comment:
I guess this is a breaking change
Could be **_attachments_** ad optional field added to the previous CreatedMessageWithContent?

Reply:
of course it's optional: https://github.com/pagopa/io-backend/pull/648/files#diff-aa6cd0613984937586c55eb6ed53d7baR816

Reply:
@gunzip 
Maybe I'm missing something

### actual message shape
```json
{
  "content": {
    "markdown": "markdown",
    "subject": "Benvenuto su IO"
  },
  "created_at": "2020-03-24T09:45:44.650Z",
  "fiscal_code": "XXXXTT00A12L719R",
  "id": "01E45YXZYAP49FC7G334ZX7NR8",
  "sender_service_id": "azure-deployc49a",
  "time_to_live": 3600
}
```

### new shape
```json
{
  "message": {
    "content": {
      "markdown": "markdown",
      "subject": "Benvenuto su IO"
    },
    "created_at": "2020-03-24T09:45:44.650Z",
    "fiscal_code": "XXXXTT00A12L719R",
    "id": "01E45YXZYAP49FC7G334ZX7NR8",
    "sender_service_id": "azure-deployc49a",
    "time_to_live": 3600
  },
  "attachments": []
}
```

this PR proposes a wrapping version of the previous message.
Isn't it?

Reply:
havent noticed that, of course it must not wrap the message

Reply:
another choice is if we should put attachments into content or aside message metadata

Reply:
So we have to modify io-functions-commons and provide a new release to point to new specs

Reply:
no we don't, those specs are good as is. the message shape here is constrained to the backend-app usage so we just have to modify the message shape here

Reply:
I don't understand. If we have to shape the message structure based on the last example, we have to modify `MessageContent` spec that has been defined in io-functions-commons. Do we have any mechanism to extend MessageContent in `api_backend.yaml` ?

Reply:
In this case, providing a new spec for `MessageContent`, i.e: `MessageContentWithAttachments`, which strategy can we adopt to inject this new spec into `CreatedMessageWithContent` ?

Reply:
we must rewrite it here

Reply:
ping:)